tcms/testcases/static/testcases/js/search.js

```diff
@@ -15,40 +15,49 @@ $(document).ready(function() {
         ajax: function(data, callback, settings) {
             var params = {};
 
-            if ($('#id_summary').val()) {
-                params['summary__icontains'] = $('#id_summary').val();
+            const summary = $('#id_summary').val();
+            if (summary) {
+                params['summary__icontains'] = summary;
```

Comment:
Do not make unrelated changes b/c it makes this PR harder to review and is out of scope. 

Revert the last commit and fix only the original issues commented in previous code review.